crates/hyperqueue/src/dashboard/ui/widgets/cpu_util_table.rs

```diff
@@ -0,0 +1,59 @@
+use std::cmp;
```

Comment:
Will this widget be used at any other place? Until it is, I suggest to move it under `screens/worker`.